test/scroll_test.rb

```diff
@@ -0,0 +1,128 @@
+require_relative "test_helper"
+
+class ScrollTest < Minitest::Test
+  def test_limit
```

Comment:
Let's remove all tests that don't involve the the scroll API. Existing functionality should already be tested.